src/display.js

```diff
@@ -0,0 +1,183 @@
+      element.id = id;
+    }
+    return element;
+  };
+
+  appendElement = (child, parent) => {
+    parent.appendChild(child);
+  };
+
+  populatePage() {
+    root.innerHTML = '';
+    const h3 = this.createElement('h3', null, null);
+    const head = this.createElement('div', ['head'], null);
+    const ul = this.createElement('ul', null, null);
+    const refresh = this.createElement(
+      'i',
+      ['refresh', 'fa', 'fa-refresh'],
+      null,
+    );
+    const divInput = this.createElement('div', ['div_input'], null);
+    const input = this.createElement('input', ['input'], 'input');
+    const enter = this.createElement('span', ['enter'], null);
+    const divClearAll = this.createElement('div', ['div_clear_all'], null);
+    const btnClear = this.createElement('button', 'clear', 'clear');
+
+    h3.innerHTML = "Today's Todo";
+    input.setAttribute('placeholder', 'Add your list...');
+    enter.innerHTML = '&crarr;';
+
+    this.appendElement(h3, head);
+    this.appendElement(refresh, head);
+    this.appendElement(input, divInput);
+    this.appendElement(enter, divInput);
+    this.appendElement(head, root);
+    this.appendElement(divInput, root);
+
+    for (let i = 0; i < tasks.taskCollection.length; i += 1) {
+      const li = this.createElement('li', null, null);
+      const chk = this.createElement('input', ['chk'], 'chk');
+      const task = this.createElement('input', ['desc'], 'desc');
+      const elipse = this.createElement(
+        'i',
+        ['elipse', 'fa', 'fa-ellipsis-v'],
+        'elipse',
+      );
+      const can = this.createElement('i', ['can', 'fa', 'fa-trash'], `${i}`);
+
+      chk.type = 'checkbox';
+      chk.checked = tasks.taskCollection[i].completed;
+      task.type = 'text';
+      task.disabled = true;
+      task.value = tasks.taskCollection[i].description;
+      if (chk.checked) {
+        task.classList.add('strike');
+      } else {
+        task.classList.remove('strike');
+      }
+      can.setAttribute('aria-hidden', 'true');
+      this.appendElement(chk, li);
+      this.appendElement(task, li);
+      this.appendElement(elipse, li);
+      this.appendElement(can, li);
+
+      this.appendElement(li, ul);
+      this.updateTask(task);
+      this.addEnterListener(task);
+
+      this.addCheckEventListener(chk, task, i);
+    }
+    this.appendElement(ul, root);
+    btnClear.innerHTML = 'Clear all completed';
+    this.appendElement(btnClear, divClearAll);
+    this.appendElement(divClearAll, root);
+
+    this.addEnterListener(input);
+    this.clearButtonListener(btnClear);
+  }
+
+  addCheckEventListener(chk, tsk, idx) {
+    chk.addEventListener('change', () => {
+      if (chk.checked) {
+        tasks.taskCollection[idx].completed = true;
+        tsk.classList.add('strike');
+        this.populatePage();
+      } else {
+        tasks.taskCollection[idx].completed = false;
+        tsk.classList.remove('strike');
+        this.populatePage();
+      }
+    });
+  }
```

Comment:
- [ ]  Great job adding the event listener on the checkbox :clap: However, I noticed that when the checkbox is ticked as completed, the change isn't reflected in the local storage so from the video below, you can see that the value of completed remains false. However, you should ensure that it changes from false to true when the checkbox is ticked and saved in the local storage. :+1: 
- [ ] When the updated array of items is stored in local storage, the user gets the correct list values after the page reloads. From the video below, you can see that when I reload the page after checking some boxes as completed, they all go back to the default so kindly work on this :wink:

[Screencast from 07-21-2023 09:55:45 AM.webm](https://github.com/wacoo/smart-todo-list/assets/102699958/21f41f12-82f1-4ae7-bffb-6b573cd45934)